internal/versions/versions.go

```diff
@@ -268,6 +271,9 @@ func (h *handler) GetReleaseImage(openshiftVersion, cpuArchitecture string) (*mo
 
 // Returns the latest OSImage entity for a specified CPU architecture
 func (h *handler) GetLatestOsImage(cpuArchitecture string) (*models.OsImage, error) {
+	if cpuArchitecture == common.MultiCPUArchitecture {
```

Comment:
Same, just some comment to make it clear.

Reply:
Sure 👍🏻